general/g.gisenv/main.c

```diff
@@ -224,6 +225,7 @@ char *parse_variable(const char *v_name, char **value)
         G_verbose_message(_("GRASS variable must be uppercase. Using '%s'."),
                           u_name);
     }
+    G_free(name);
```

Comment:
I think that you should also add a `G_free` at line 218 before the `G_fatal_error`

Reply:
It wouldn't be bad, but introduce unnecessary code. `G_fatal_error` will print an error message and `exit` the process. Thus, in-memory will be freed safely and quickly by the system.